StRoot/StRefMultCorr/Param.h

```diff
@@ -481,10 +471,96 @@ const string mParamStr_ref1[nID_ref1][nSet_ref1] = {
+    "1.2680,-0.13,0.0593,0.0179,-0.0879,0,0.120,0.00714",   // Trigger efficiency
+    "0.,0."                      // Luminosity correction
+  },
+
+  // mParameterIndex = 2
+  // Run 20 Au+Au 5.75 GeV (sqrt(s_NN)=3.5 GeV), Trigger ID = 720000
+  {
+    "2020:5.75:20355020,21045011:198,202",         // Year, energy, run start end, Vz range
+    "6,8,12,16,21,28,37,47,59,73,89,108,128,153,181,216",  // Multiplicity values for the centrality classes
+    "107",                        // Normalization start (of Glauber to the data)
+    "0.,0.,0.,0.,0.,0.,0.,0.",   // Vz correction parameters (switched to new scheme, parameters defined near the shape corrections)
+    "1.22,-0.137,0.0532,0.00845,-0.0690,0,0.1,0.00546",   // Trigger efficiency
+    "0.,0."                      // Luminosity correction
+  },
+
+  // mParameterIndex = 3
+  // Run 19 Au+Au 7.3 GeV (sqrt(s_NN)=3.9 GeV), Trigger ID = 1
+  {
+    "2019:7.3:20107029,20169055:198,202",         // Year, energy, run start end, Vz range
+    "7,9,13,18,24,32,41,52,65,81,98,119,141,168,198,235",  // Multiplicity values for the centrality classes
+    "141",                        // Normalization start (of Glauber to the data)
+    "0.,0.,0.,0.,0.,0.,0.,0.",   // Vz correction parameters (switched to new scheme, parameters defined near the shape corrections)
+    "1.278,-0.1635,0.0547,0.217,-0.0734,0,0.222,0.00481",   // Trigger efficiency
+    "0.,0."                      // Luminosity correction
+  },
+
+  // mParameterIndex = 4
+  // Run 20 Au+Au 7.3 GeV (sqrt(s_NN)=3.9 GeV), Trigger ID = 730000
+  {
+    "2020:7.3:21035003,21036013:198,202",         // Year, energy, run start end, Vz range
+    "6,9,13,17,23,31,40,51,64,80,97,118,141,167,198,236",  // Multiplicity values for the centrality classes
+    "141",                        // Normalization start (of Glauber to the data)
+    "0.,0.,0.,0.,0.,0.,0.,0.",    // Vz correction parameters (switched to new scheme, parameters defined near the shape corrections)
+    "1.017,0.0609,0.0744,-0.106,-0.003,0,0.0824,-3.91e-4",   // Trigger efficiency
+    "0.,0."                       // Luminosity correction
+  },
+
+  // mParameterIndex = 5
+  // Run 20 Au+Au 31.2 GeV (sqrt(s_NN)=7.7 GeV), Trigger ID = 770000
+  {
+    "2020:7.3:21028011,21029036:198,202",         // Year, energy, run start end, Vz range
+    "10,15,21,28,38,49,63,79,98,119,142,169,200,233,271,315",  // Multiplicity values for the centrality classes
+    "160",                        // Normalization start (of Glauber to the data)
+    "0.,0.,0.,0.,0.,0.,0.,0.",    // Vz correction parameters (switched to new scheme, parameters defined near the shape corrections)
+    "9.63078e-1,6.54124e-02,4.68589e-02,-2.43086e-02,8.15939e-03,0,4.50613e-02,-1.14802e-03",   // Trigger efficiency
+    "0.,0."                       // Luminosity correction
+  }
+};
+
+
```

Comment:
Second run ID range of 5.75 GeV Run 20
```suggestion
  // mParameterIndex = 6
  // Run 20 Au+Au 5.75 GeV (sqrt(s_NN)=3.5 GeV), Trigger ID = 720000
  {
    "2020:5.75:21044023,21045011:198,202",         // Year, energy, run start end, Vz range
    "6,8,12,16,21,28,37,47,59,73,89,108,128,153,181,216",  // Multiplicity values for the centrality classes
    "107",                        // Normalization start (of Glauber to the data)
    "0.,0.,0.,0.,0.,0.,0.,0.",   // Vz correction parameters (switched to new scheme, parameters defined near the shape corrections)
    "1.22,-0.137,0.0532,0.00845,-0.0690,0,0.1,0.00546",   // Trigger efficiency
    "0.,0."                      // Luminosity correction
  },
  
```